.github/workflows/dependency_merge.yml

```
@@ -51,10 +37,10 @@ jobs:
 
       - name: Wait for other status checks to Pass
         id: waitforstatuschecks
-        uses: lewagon/wait-on-check-action@v1.3.3
+        uses: lewagon/wait-on-check-action@v1.3.1
```

Can we keep it latest?
```suggestion
        uses: lewagon/wait-on-check-action@v1.3.3
```